insights/parsers/cloud_cfg.py

```diff
@@ -0,0 +1,24 @@
+"""
+CloudCfg - file ``/etc/cloud/cloud.cfg``
+========================================
```

Comment:
@psachin please make sure you include the file `docs/shared_parsers/cloud_cfg.rst` file to include this parser in the documentation.

Reply:
Hey @bfahr Sorry I missed your comment. The doc is added now.